homeassistant/components/starlink/coordinator.py

```diff
@@ -60,7 +63,10 @@ async def _async_update_data(self) -> StarlinkData:
                 location = await self.hass.async_add_executor_job(
                     location_data, self.channel_context
                 )
-                return StarlinkData(location, *status)
+                sleep = await self.hass.async_add_executor_job(
+                    get_sleep_config, self.channel_context
+                )
```

Comment:
Could we gather the calls to `status_data`, `location_data` and `get_sleep_config` or do they need to be called sequentially?

Reply:
I mean use [`asyncio.gather`](https://docs.python.org/3/library/asyncio-task.html#asyncio.gather) to run the three corutines in parallel

Reply:
Dxone, thanks!